docs/kyma/docs/037-gs-installation-overrides.md

```diff
@@ -0,0 +1,168 @@
+In one of it's templates there's a following fragment (shortened):
+```
+spec:
+  containers:
+  - name: {{ .Chart.Name }}
+	args:
+	  - "/connectorservice"
+	  - '--appName={{ .Chart.Name }}'
+	  - "--domainName={{ .Values.global.domainName }}"
+	  - "--tokenExpirationMinutes={{ .Values.deployment.args.tokenExpirationMinutes }}"
+```
+
+The following fragment of `values.yaml` file in `connector-service` chart defines default value for `tokenExpirationMinutes`:
+```
+deployment:
+  args:
+    tokenExpirationMinutes: 60
+```
+
+If you want to override this value, for example to "90", do as follows:
+- Create a ConfigMap in kyma-installer namespace, labelled with: `installer:overrides` (or reuse existing one)
+- Add an entry `application-connector.connector-service.deployment.args.tokenExpirationMinutes: 90` to the map.
+
+Notice that the user-provided override key is now composed from two parts:
+  - Chart "path" inside top-level `core` chart: **application-connector.connector-service**
+  - Original template value reference from the chart (without .Values. prefix): **deployment.args.tokenExpirationMinutes**
+Once the installation starts, Installer generates overrides based on the map entries and value of "90" will be used instead of default "60" from the chart `values.yaml` file.
+
+
+#### Global overrides
+
+Some important parameters are usually shared across the charts.
+Helm convention to provide these is by using `global` override key.
+
+For example, to define `global.domain` override, just use "global.domin" as the name of the key in ConfigMap/Secret for Installer.
+Once the installation starts, Installer is merging all map entries and collect all global entries together (under `global` top-level key) for the purpose of installation.
+
+
+## Merging and conflicting entries
+
+When installer encounters two overrides with the same key prefix, it tries to merge them.
+If both of them represent a map (they have nested sub-keys), their nested keys are recursively merged.
+If at least one of keys points to a final value,  Installer performs the merge in non-deterministic order, so either one or the other override is rendered to the final YAML.
+Considering that, **avoid overrides having the same keys for final values**.
+
+### Example of non-conflicting merge:
+
+Having two overrides with a common *key prefix* ("a.b"):
+
+```
+"a.b.c": "first"
+"a.b.d": "second"
+```
+
+Installer yields correct output:
+
+```
+a:
+  b:
+    c: first
+    d: second
+```
+
+### Example of conflicting merge:
+
+Having two overrides with *the same* key ("a.b"):
+
+```
+"a.b": "first"
+"a.b": "second"
+```
+
+Installer yields either:
+
+```
+a:
+  b: "first"
+```
+
+Or (due to non-deterministic merge order):
+
+```
+a:
+  b: "second"
+```
+
+
+## Values and types
+
+Installer generally recognizes all override values as **strings** and internally render overrides to Helm as YAML stream with only **string** values.
```

Comment:
Installer generally recognizes all override values as strings. It internally renders overrides to Helm as a yaml stream with only string values.

There is one exception to this rule with respect to handling booleans:  

The system converts "true" or "false" strings that it encounters to a corresponding boolean value (true/false).